docs/mimxrt/quickref.rst

```diff
@@ -301,13 +301,16 @@ There are up to four hardware SPI channels that allow faster transmission
 rates (up to 30Mhz).  Hardware SPI is accessed via the
 :ref:`machine.SPI <machine.SPI>` class and has the same methods as software SPI above::
 
-    from machine import SPI
+    from machine import SPI, Pin
 
-    spi = SPI(0, 10000000)
+    cs_pin = Pin(6, Pin.OUT, value=1)
     spi.write('Hello World')
 
 For the assignment of Pins to SPI signals, refer to
 :ref:`Hardware SPI pinout <mimxrt_spi_pinout>`.
+The keyword option cs=n can be used to enable the cs pin 0 or 1 for an automatic cs signal. The
+default is cs=0. Using cs=-1 the automatic cs signal is not created. 
+In that case, cs has to be set by the script. Clearing that assignment requires a power cycle.
```

Comment:
Maybe the default could be changed to `cs=-1`.  This would match other ports which don't have hardware CS.

Reply:
I considered that forth and back. the default of cs=0 is compatible to the existing mimxrt behaviour, cs=-1 is compatible to the other boards. Either choice is easy to make.

Reply:
Set to cs=-1 as default.

Reply:
Great, thanks!